testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -298,6 +309,104 @@ impl TxEmitter {
         })
     }
 
+    pub async fn load_dd_account(&self, instance: &Instance) -> Result<AccountData> {
+        let client = instance.json_rpc_client();
+        let mint_key: Ed25519PrivateKey = generate_key::load_key(DD_KEY);
+        let mint_key_pair: KeyPair<Ed25519PrivateKey, Ed25519PublicKey> = KeyPair::from(mint_key);
+        let address = libra_types::account_address::from_public_key(&mint_key_pair.public_key);
+        let sequence_number = query_sequence_numbers(&client, &[address])
+            .await
+            .map_err(|e| {
+                format_err!(
+                    "query_sequence_numbers on {:?} for dd account failed: {}",
+                    client,
+                    e
+                )
+            })?[0];
+        Ok(AccountData {
+            address,
+            key_pair: mint_key_pair.clone(),
+            sequence_number,
+        })
+    }
+
+    pub async fn load_vasp_account(&self, instance: &Instance) -> Result<AccountData> {
+        let client = instance.json_rpc_client();
+        let mint_key: Ed25519PrivateKey = generate_key::load_key(VASP_KEY);
+        let mint_key_pair: KeyPair<Ed25519PrivateKey, Ed25519PublicKey> = KeyPair::from(mint_key);
+        let address = libra_types::account_address::from_public_key(&mint_key_pair.public_key);
+        let sequence_number = query_sequence_numbers(&client, &[address])
+            .await
+            .map_err(|e| {
+                format_err!(
+                    "query_sequence_numbers on {:?} for dd account failed: {}",
+                    client,
+                    e
+                )
+            })?[0];
+        Ok(AccountData {
+            address,
+            key_pair: mint_key_pair.clone(),
+            sequence_number,
+        })
+    }
+
+    pub async fn mint_faucet_account(
```

Comment:
faucet_account at this point becomes a bit ambiguous, as dd account is certainly not a faucet account.
Perhaps rename this function to `get_money_source`?